src/mesh/mesh-pb-constants.h

```diff
@@ -18,6 +18,9 @@
 #define MAX_RX_TOPHONE 32
 #endif
 
+/// Verify baseline assumption of node size
+static_assert(sizeof(meshtastic_NodeInfoLite) <= 192, "NodeInfoLite size is larger than expected");
```

Comment:
Shall we make the comment a bit more clear? Why is it expected to be 192? Maybe something like "NodeInfoLite size increased, reconsider `MAX_NUM_NODES`".